src/clincoded/schemas/curatorHistory.json

```diff
@@ -14,7 +14,7 @@
     ],
     "properties": {
         "schema_version": {
-            "default": "3"
+            "default": "5"
```

Comment:
Should schema version be "4" here? Let me know what you think!

Reply:
This is 5 because this is branched off of the pubmed notes branch. The other branch upgrades to 4 and this branch upgrades to 5.

Reply:
Sweet, thanks for clarifying!